### PR TITLE
Make compliant to cakephp 3 coding standards

### PR DESCRIPTION
**Pull request by kevinquinnyo**
Tests passed after changes.  Then formatted tests themselves, tests still passing.
reference issue #47

### CODE REVIEW

**Comment by kevinquinnyo** [comment]
FYI this failed because it breaks compliance with the coding standard the Travis CI build is using.  What should be done to correct that?

**Comment by ADmad** [comment]
Update the relevant statement in .travis.yml file at root of plugin.

**Comment by ADmad** [comment]
Current travis config is pretty outdated and still uses PEAR to get phpcs. You can refer to [this](https://github.com/FriendsOfCake/travis/blob/cake3/templates/.travis.yml) template to update the config to use composer for getting phpcs.

File: src/Auth/CookieAuthenticate.php [hunk 1]
```diff
@@ -14,113 +14,117 @@
  * Provides the ability to authenticate using COOKIE
  *
  * {{{
```
Comment (ADmad) [inline src/Auth/CookieAuthenticate.php]: Please also replace the code fence `{{{` and `}}}`  with markdown standard ```

**Comment by kevinquinnyo** [comment]
@ADmad Ah, got it!  Thanks, I'll fix that and resubmit.

**Comment by davidyell** [comment]
:+1: Good work, I wondered about this today.

File: src/Auth/TokenAuthenticate.php [hunk 2]
```diff
@@ -13,158 +13,163 @@
...
+            'header' => 'X-ApiToken',
+            'fields' => ['token' => 'token', 'password' => 'password'],
+            'continue' => false,
+            'unauthorized' => 'Cake\Network\Exception\BadRequestException'
+        ]);
+
+        $this->config($config);
+
+        if (empty($this->_config['parameter']) &&
+        empty($this->_config['header'])
```
Comment (ADmad) [inline src/Auth/TokenAuthenticate.php]: This line should be intended. Weird that phpcs doesn't complain about it.

File: src/Auth/TokenAuthenticate.php [hunk 3]
```diff
@@ -13,158 +13,163 @@
...
+    public function getUser(Request $request)
+    {
+        if (!empty($this->_config['header'])) {
+            $token = $request->header($this->_config['header']);
+            if ($token) {
+                return $this->_findUser($token);
+            }
+        }
+        if (!empty($this->_config['parameter']) &&
+        !empty($request->query[$this->_config['parameter']])
```
Comment (ADmad) [inline src/Auth/TokenAuthenticate.php]: Indentation needed

**Comment by ADmad** [comment]
Thanks @kevinquinnyo

**Pull request merged**